content/applications/finance/fiscal_localizations/argentina.rst

```diff
@@ -872,6 +872,103 @@ The menu also displays critical information related to these operations, such as
+
+.. image:: argentina/l10nar-fiscal-fields-ar-ecommerce.png
+   :align: center 
+   :alt: l10n_ar eCommerce module
+
+This module enables the features and configurations to:
+
+- Clients being able to create online accounts for *eCommerce* purposes.
+- Support for required fiscal fields in the *eCommerce* application.
+- Receive payments for sale orders online.
+- Generate electronic documents from the *eCommerce* application.
+
+.. note::
+   The Argentinian eCommerce module is dependent on the previous installation of the 
+**Invoicing** or **Accounting** Apps
+   
+Configurations
+~~~~~~~~~~~~~~
+
+Once all of the configurations are made for the Argentinian :ref:`electronic invoice 
+<argentina/configure-your-company>` flow, it is also needed to complete certain configurations for 
+the eCommerce flow to be integrated.
+
+Client Account Registration
+***************************
+
+It is possible to configure your website to select from the next options regarding client accounts:
+
+- **Optional**: It allows guests in the Website to register directly in the confirmation email from 
+their order
+- **Disabled (buy as guest)**: It allows guests to buy and pay without an account created
+- **Mandatory (no guest checkout)**: It is obligatory to create an account to be able to buy and pay 
+in the eCommerce
+
+To access this configuration go to :menuselection:`Website --> Configuration --> Settings --> Shop - Checkout Process`.
+
+.. note::
+   It is recommended to use the **Optional** or **Mandatory** options, since this will save the 
+contact and their fiscal information for future purchases.
+   
+Automatic Invoice
+*****************
+
+Configure your website so it is possible to generate electronic documents in the sale process by 
+going to :menuselection:`Website --> Configuration --> Settings --> Invoicing` and activating the 
+:guilabel:`Automatic Invoice` feature, which will automatically generate the required electronic 
+documents when the online payment is confirmed.
+
+.. image:: argentina/l10nar-automatic-invoicing-ecommerce.png
+   :align: center 
+   :alt: Feature activated to invoice automatically
+   
+Since an online payment needs to be confirmed for the :guilabel:`Automatic Invoice` feature to 
+generate the document, a :guilabel:`payment provider` should be configured for the related website.
+
+.. note::
+   Review the :doc:`../payment_providers` documentation for information on which payment providers
+   are supported in Odoo, and how to configure them.
+   
+Products
+********
+
+To be able to invoice your products at the point of an online payment confirmation, go to the desired 
+product and select the :guilabel:`Invoicing Policy` to :guilabel:`Ordered quantities`.
+
+Also, it is important to define the desired :guilabel:`Customer taxes`, since this is a required field 
+when creating an invoice that will be part of an electronic invoice flow.
```

Comment:
```suggestion
To allow your products to be invoiced when an online payment is confirmed, navigate to the desired
product from :menuselection:`Website --> eCommerce --> Products` and on the :guilabel:`General
Information` tab, set the :guilabel:`Invoicing Policy` to :guilabel:`Ordered quantities`.

Also, it is important to define the desired :guilabel:`Customer Taxes`, since this is a required
field when creating an invoice that will be part of an electronic invoice flow.
```